logpolicy/logpolicy.go

```diff
@@ -769,6 +769,11 @@ func NewLogtailTransport(host string, netMon *netmon.Monitor, logf logger.Logf)
 	}
 	tr.DialContext = MakeDialFunc(netMon, logf)
 
+	// We're uploading logs ideally infrequently, with specific timing that will
+	// change over time. Try to keep the connection open, to avoid repeatedly
+	// paying the cost of TLS setup.
+	tr.IdleConnTimeout = time.Hour
+
```

Comment:
Do we want to explicitly set `tr.DisableKeepAlives ` to false with a comment that we want to make use of HTTP keep-alives, which allows us to make back-to-back requests on the same connection?

Reply:
Also, did we want to disable TCP keep alives in the same PR?

Reply:
It's set to false by default, I can't currently imagine a world where that default is going to change

Reply:
They already are, as a result of the netknob, but it happens in the dialer

Reply:
My intent wasn't so much about changing the value, but documenting that we rely on the implicit default for performance reasons.